yt/data_objects/data_containers.py

```diff
@@ -754,6 +755,10 @@ def create_firefly_object(
             A flag to optionally print the fields that are available, in the
             dataset but were not explicitly requested to be tracked.
 
+        match_any_particle_types : boolean
+            A flag to specify that ambiguous fields in fields_to_include
+            should match all relevant particle groups instead of raising an
+            error.
```

Comment:
I think the language could be a little clearer here. How about:
```suggestion
        match_any_particle_types : boolean
            If True, when any of the fields_to_include match multiple particle groups then 
            the field will be added for all matching particle groups. If False, an error is raised 
            when encountering an ambiguous field. Default is True.
```